src/sentry/scim/endpoints/utils.py

```diff
@@ -75,15 +98,26 @@ class SCIMEndpoint(OrganizationEndpoint):
     def add_cursor_headers(self, request, response, cursor_result):
         pass
 
-    def list_api_format(self, request, total_results, results):
+    def list_api_format(self, results, total_results, start_index):
         return {
             "schemas": [SCIM_API_LIST],
             "totalResults": total_results,  # TODO: audit perf of queryset.count()
-            "startIndex": int(request.GET.get("startIndex", 1)),  # must be integer
+            "startIndex": start_index,
             "itemsPerPage": len(results),  # what's max?
             "Resources": results,
         }
 
+    def get_query_parameters(self, request):
```

Comment:
I wanted to do this in a `convert_args` function so that the params are available to all SCIM endpoints, but that gets whacky with the multiple inheritances used in the SCIM routes, so thus a method on the class.